pkg/data/dashboard/namespace.go

```diff
@@ -11,14 +12,26 @@ import (
 )
 
 func addCattleGlobalNamespaces(ctx context.Context, k8s kubernetes.Interface) error {
+	if features.Fleet.Enabled() {
+		_, err := k8s.CoreV1().Namespaces().Get(ctx, "fleet-local", metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
```

Comment:
What if it isn't a `NotFound` error?

Reply:
Yeah that's not handled, I'll change that.